docs/src/base-profile-transformations.md

```diff
@@ -117,44 +136,56 @@ entry:
 }
 ```
 
-We note that we successfully have eliminated loops, arithmetic operations, dynamic qubit allocation and alias counting - all operations which are not supported by the base profile.
+We note that we successfully have eliminated loops, arithmetic operations,
+dynamic qubit allocation and alias counting - all operations which are not
+supported by the base profile.
 
-## Goal
+### Goal
```

Comment:
@troelsfr Should we be showing QIR -> QIR here? I think BC -> BC would be a preferred usage pattern. If they want to see the QIR it can be generated. This really applies to most usages of `qat` in the docs.

Reply:
I am not sure I follow your question. Could you help me out a bit with what you are after?

As a side note, QAT can do both QIR->QIR and at least output BC. If recall correctly it can also read it (but it is a long time since wrote that part of the code), so it should be able to do (QIR|BC) -> (QIR|BC). Hence, when I talk about QIR in the text, I really mean either QIR or BC.